logstash/values.yaml

```diff
@@ -0,0 +1,219 @@
+---
+replicas: 1
+
+# Allows you to add any config files in /usr/share/logstash/config/
+# such as logstash.yml and log4j2.properties
+logstashConfig: {}
+#  logstash.yml: |
+#    key:
+#      nestedkey: value
+#  log4j2.properties: |
+#    key = value
+
+# Allows you to add any pipeline files in /usr/share/logstash/pipeline/
+logstashPipeline: {}
+#  ls.conf: |
+#    input {
+#      exec {
+#        command => "uptime"
+#        interval => 30
+#      }
+#    }
+#    output { stdout { } }
+
+# Extra environment variables to append to this nodeGroup
+# This will be appended to the current 'env:' key. You can use any of the kubernetes env
+# syntax here
+extraEnvs: []
+#  - name: MY_ENVIRONMENT_VAR
+#    value: the_value_goes_here
+
+# A list of secrets and their paths to mount inside the pod
+secretMounts: []
+
+image: "docker.elastic.co/logstash/logstash"
+imageTag: "7.4.0"
+imagePullPolicy: "IfNotPresent"
+imagePullSecrets: []
+
+podAnnotations: {}
+
+# additionals labels
+labels: {}
+
+logstashJavaOpts: "-Xmx1g -Xms1g"
+
+resources:
+  requests:
+    cpu: "100m"
+    memory: "2Gi"
+  limits:
+    cpu: "1000m"
+    memory: "2Gi"
+
+initResources: {}
+  # limits:
+  #   cpu: "25m"
+  #   # memory: "128Mi"
+  # requests:
+  #   cpu: "25m"
+  #   memory: "128Mi"
+
+sidecarResources: {}
+  # limits:
+  #   cpu: "25m"
+  #   # memory: "128Mi"
+  # requests:
+  #   cpu: "25m"
+  #   memory: "128Mi"
+
+volumeClaimTemplate:
+  accessModes: [ "ReadWriteOnce" ]
+  resources:
+    requests:
+      storage: 1Gi
+
+rbac:
+  create: false
+  serviceAccountName: ""
```

Comment:
Curious, does Logstash need a kubernetes service account for some purpose?

Reply:
Also a leftover of Elasticsearch copy/paste. I don't know yet if we have a specific need for Logstash yet, but all other charts are managing ServiceAccount

Reply:
So while Logstash may not have specific need of a ServiceAccount, it seems to be really usefull for people implementing [Pod Security Policies](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) in their K8S cluster (see https://github.com/elastic/helm-charts/issues/61#issuecomment-463784565).